### PR TITLE
[ADD]l10n_es_SII: and refactoring pms documents

### PR DESCRIPTION
**Pull request by DarioLodeiros**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please don't use uppercase on module name, and you are repeating some of the information already present in `l10n_es_aeat`. Anyway, this module is not very suitable to be here in this repo. It's not the same as the previous "localized" one, that was totally related with PMS.

**Comment by DarioLodeiros** [comment]
the name of the module is changed, but as for the comment of the repeated information, I don't realize what information I am duplicating ¿?

File: pms_l10n_es_sii/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    vat = fields.Char(
+        readonly=False,
+        store=True,
+        compute="_compute_vat",
+    )
+    aeat_identification_type = fields.Selection(
```
Comment (pedrobaeza) [inline pms_l10n_es_sii/models/res_partner.py]: This is already present in `l10n_es_aeat`

Comment (DarioLodeiros) [reply pms_l10n_es_sii/models/res_partner.py]: aeat_identification_type field?, this module converts the field to compute, in l10n_es_aeat the field is a normal selection field

Comment (pedrobaeza) [reply pms_l10n_es_sii/models/res_partner.py]: And that's a mistake that is influencing in normal SII operations. And as said, this module shouldn't go here.

Comment (DarioLodeiros) [reply pms_l10n_es_sii/models/res_partner.py]: Ok I understand. I could make the module independent of the pms to simply integrate the document numbers with the aeat document types. And with that PR to l10n_es, but I don't understand the problem of computing the fields, I think that the logic of the compute method itself takes care not to spoil any base behavior (it's a simple "emulation" of the onchange). If you see ok the compute, I do the PR to l10n_es, if not I can keep it in an internal repository without problem.
Thanks for the review!

**Comment by pedrobaeza** [comment]
Yes, please, try to keep `l10n_es_pms` minimal for the PMS related legislation, and the rest of delicate things can be something custom for your instances IMO.

**Comment by DarioLodeiros** [comment]
100%, thanks!

**Comment by DarioLodeiros** [comment]
https://github.com/OCA/l10n-spain/pull/2520

**Pull request merged**